doc/development/tutorials/examples/recipe.py

```diff
@@ -87,7 +87,7 @@ def generate(self, docnames=None):
         # first letter of the recipe as a key to group thing
         #
         # name, subtype, docname, anchor, extra, qualifier, description
-        for name, dispname, typ, docname, anchor, _ in recipes:
+        for _name, dispname, typ, docname, anchor, _ in recipes:
```

Comment:
Where this leading underscore convention comes from?

Reply:
While it's not mentioned in PEP8, it's a *very* common convention in Python. It's also a convention in JavaScript, typescript, Haskell, Rust, Ruby, Go, and probably others

Reply:
In this line, the last element is named as "_". It's also a well-used technique, I think. Which is better? If we'll switch to your proposal, do we have to fix it too?

Reply:
Note: the last element of `recipes` is a "priority" value.

Reply:
> In this line, the last element is named as "_". It's also a well-used technique, I think. Which is better? If we'll switch to your proposal, do we have to fix it too?

Using a single underscore is also a common convention. Personally I prefer the leading underscore because its more explicit about *what* you're ignoring

Reply:
done